### PR TITLE
165406920 implement getting all followings and followers

### PR DESCRIPTION
**Pull request by Raymond-Osy**
#### Description
This is an API endpoint that gets all the users been followed by an authenticated user

#### Type of change
- [ ] Bug Fix (a non-breaking change which fixes an issue)
- [x] New Feature (a non-breaking change that adds functionality)
- [ ] New Chore (a non-breaking change that adds functionality)
- [ ] Breaking Change (a fix or feature which would cause existing functionality not to work as 

#### How Has This Been Tested?
- [ ] End to End
- [x] Integration

#### Checklist:
- [ ] End to end
- [x] Integration

#### PT
[#165406920](https://www.pivotaltracker.com/story/show/165406920)

#### Postman Snapshot
All followings
Screenshot 2019-04-24 at 3 03 00 PM

All followers
Screenshot 2019-04-24 at 3 03 15 PM

### CODE REVIEW

File: src/controllers/follower.js [hunk 1]
```diff
@@ -48,9 +40,98 @@ export const followUser = async (req, res) => {
...
+      include: [{
+        model: Follower,
+        as: 'followers',
+        include: {
+          model: User,
+          as: 'following',
+          attributes: ['id', 'fullName', 'email']
+        } }],
+      attributes: ['id', 'fullName']
+    })).toJSON();
```
Comment (Temmyogunbo) [inline src/controllers/follower.js:123]: Why are you using `toJSON` method?

Comment (Raymond-Osy) [reply src/controllers/follower.js:123]: The team gave me feedback to only output the details of the followers, id, Name, and email so I needed to use the map method to return just these details. That was the reason.

**Review by chikeozulumba — COMMENTED** [review]
Nice Work @Raymond-Osy 
LGTM

**Review by micah-akpan — COMMENTED** [review]
LGTM

**Review by Temmyogunbo — APPROVED** [review]

**Pull request merged**